contracts/dispatcher/src/state/mod.rs

```diff
@@ -17,7 +17,7 @@ pub struct DispatchLog {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
-pub struct Config {
+pub(crate) struct Config {
     // Time duration in hours defining the periods of time this instance is awaken
     pub cadence_hours: u16,
```

Comment:
[out of scope but relatively simple and cheap to include in this PR] please define a `type` for cadence hours and use it anywhere